neuromation/cli/formatters/images.py

```diff
@@ -133,3 +135,20 @@ def commit_finished(self, data: ImageCommitFinished) -> None:
 
     def close(self) -> None:
         self._printer.close()
+
+
+class BaseImagesFormatter:
+    @abc.abstractmethod
+    def __call__(self, images: Iterable[RemoteImage]) -> Iterable[str]:
+        raise NotImplementedError
+
+
+class ShortImagesFormatter(BaseImagesFormatter):
+    def __call__(self, images: Iterable[RemoteImage]) -> Iterable[str]:
+        return (str(image) for image in images)
+
+
+class LongImagesFormatter(BaseImagesFormatter):
+    def __call__(self, images: Iterable[RemoteImage]) -> Iterable[str]:
+        rows = [[str(image), image.as_docker_url] for image in images]
```

Comment:
The last neat: please add the table header, like for `neuro ps` command.

Reply:
What would be good headers? 'IMAGE URL', 'DOCKER URL'?

Reply:
I guess `Neuro URL` and `Docker URL` (both capitalized) -- to reflect the fact that the first value should be used with `neuro`, the second is for `docker`.